app/grants/templates/grants/explorer.html

```diff
@@ -359,6 +359,10 @@ <h1 class="font-title-lg font-weight-semibold mt-4 mb-2">{% trans "This is not t
                     </button>
                   </div>
                 </div>
+
+                <div class="col-12 mb-3 mx-auto col-md-6 text-center text-center" v-show="lock">
+                  <img class="w-100" src="https://s.gitcoin.co/static/v2/images/loading_v2.f27849139c28.gif">
```

Comment:
Absolute URLs should be hardcoded. Please us "{% static ... %}"

Reply:
Changed this to "{% static %}" format.